api/image_processor.py

```diff
@@ -1,9 +1,22 @@
+import time
+import os
+
 from PIL import Image, ImageFilter
+from nudenet import NudeClassifier
+
+classifier = NudeClassifier()
 
 
 class ImageProcessor:
     @staticmethod
-    def blur(image: Image) -> Image:
+    def process(image: Image) -> Image:
         # Blurring the image
-        return image.filter(ImageFilter.GaussianBlur(25))
+        image_path = 'temp_{}.jpeg'.format(time.time())
+        image.save(image_path)
+        if classifier.classify(image_path)[image_path]['unsafe'] > 0.5:
```

Comment:
I think it would be better to have the classification threshold to be stored as a class member variable that can be assigned when the class is initialized.
We can have the method not be static as well.
This will allow for more freedom in customizing the class and its abilities in the future.

Reply:
I'm glad you think so! I will go ahead and make the changes you suggest.